datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```
@@ -1,30 +1,30 @@
-adodbapi==2.6.0.7; sys_platform == "win32"
+adodbapi==2.6.2.0; sys_platform == "win32"
 aerospike==4.0.0; sys_platform != "win32" and sys_platform != "darwin" and python_version < "3.0"
 aerospike==6.0.0; sys_platform != "win32" and sys_platform != "darwin" and python_version > "3.0"
 aws-requests-auth==0.4.3
 beautifulsoup4==4.9.3; python_version < "3.0"
 beautifulsoup4==4.10.0; python_version > "3.0"
 binary==1.0.0
 boto3==1.17.112; python_version < "3.0"
-boto3==1.18.48; python_version > "3.0"
+boto3==1.19.12; python_version > "3.0"
 boto==2.49.0
 botocore==1.20.112; python_version < "3.0"
-botocore==1.21.48; python_version > "3.0"
+botocore==1.22.12; python_version > "3.0"
 cachetools==3.1.1
 clickhouse-cityhash==1.0.2.3
 clickhouse-driver==0.2.0; python_version < "3.0"
-clickhouse-driver==0.2.1; python_version > "3.0"
+clickhouse-driver==0.2.2; python_version > "3.0"
 contextlib2==0.6.0; python_version < "3.0"
 cryptography==3.3.2; python_version < "3.0"
 cryptography==3.4.8; python_version > "3.0"
 cx-oracle==7.2.0
 ddtrace==0.32.2; sys_platform == "win32" and python_version < "3.0"
 ddtrace==0.53.2; sys_platform != "win32" or python_version > "3.0"
-dnspython==1.16.0
+dnspython==2.1.0
```

```suggestion
dnspython==1.16.0; python_version < "3.0"
dnspython==2.1.0; python_version > "3.0"
```
Per the agent build, it looks like dnspython 2.1.0 is not availabled for python 2 so we'll have to pin it https://www.dnspython.org/news/2.1.0/

---

I ended up removing this change because one of the py38 unit tests was breaking

---

Sounds good, do you think we should add it to the ignore list?

---

I'll add a card to the backlog to investigate upgrading dnspython to 2.0.0+